documentation/User-Documentation/Notifications.md

```diff
@@ -151,104 +215,76 @@ Shows the notification using a title.
-    # Test Categories
-    test_suppress:
-        io: suppress
-
-    test_chat:
-        io: chat
-        sound: BLOCK_CHEST_CLOSE
-
-    test_advancement:
-        io: advancement
-        sound: BLOCK_CHEST_CLOSE
-        frame: challenge # challenge|goal|task
-        icon: map
-
-    test_actionbar:
-        io: actionbar
-        sound: BLOCK_CHEST_CLOSE
-
-    test_bossbar:
-        io: bossbar
-        sound: BLOCK_CHEST_CLOSE
-        #barFlags: create_fog,darken_sky,play_boss_music
-        barColor: purple # blue|green|pink|purple|red|white
-        progress: 0.0 # 0.0 - 1.0
-        style: solid # segmented_10|segmented_12|segmented_20|segmented_6|solid
-        stay: 70
-
-    test_title:
-        io: title
-        sound: BLOCK_CHEST_CLOSE
-        fadeIn: 10
-        stay: 70
-        fadeOut: 20
-        #subtitle:
-
-    test_subtitle:
-        io: subtitle
-        sound: BLOCK_CHEST_CLOSE
-        fadeIn: 10
-        stay: 70
-        fadeOut: 20
-```
+<h3>soundplayeroffset:</h3>
+This option can be a number or a vector.
 
-And the following events run in order:
-```YAML
-# Test of Category
-notify_cat_suppress: notify Test Notify category:test_suppress
-notify_cat_chat: notify Test Notify category:test_chat
-notify_cat_advancement: notify Test Notify category:test_advancement
-notify_cat_actionbar: notify Test Notify category:test_actionbar
-notify_cat_bossbar: notify Test Notify category:test_bossbar
-notify_cat_title: notify Test Notify category:test_title
-notify_cat_subtitle: notify Test Notify category:test_subtitle
-
-# Test of Category + some custom
-notify_catcus_title_sub: notify Test Notify category:test_title subtitle:"My SubTitle"
-notify_catcus_bossbar_red: notify Test Notify category:test_bossbar barColor:red sound:ENTITY_BAT_TAKEOFF style:segmented_10 progress:0.3
-
-# Test of totally custom, needs an io
-notify_cus_bossbar_green: notify Test Notify io:bossbar barColor:green stay:120
-notify_cus_advancement: notify Test Notify io:advancement icon:hopper frame:goal
-```
+**Number**:    
 
-A video can be found [here](https://www.youtube.com/watch?v=mmEfIXp4dxA)
+The location the sound will be played at is moved away from the player towards the `soundlocation` using the number from
+`soundplayeroffset`.
+The sound will be at the actual location if the player is closer to the soundlocation then the offset would allow.
 
-## Example 2 - Bossbar Countdown
-Assuming the following events run in order:
-```YAML
-notify_cus_bossbar_countdown1: notify Countdown Test Bossbar io:bossbar stay:120 countdown:10 progress:1
-notify_cus_bossbar_countdown2: notify Countdown Test Bossbar io:bossbar stay:120 countdown:120 progress:1
-```
+??? info "Visual Explanation"  
+    ![offset image](../media/content/User-Documentation/Notifications/offset.png)
+    
+    This shows how the sound will be placed at the `soundlocation` if the distance between the player and the `soundlocation`
+    is smaller then the `playeroffset`:    
```

Comment:
The soundplayeroffset in the second picture is strange

Reply:
sound player offset needs to go through the soundlocation

Reply:
I have replaced these images with corrected and higher quality versions.